service/user-read/composer.json

```diff
@@ -21,13 +21,13 @@
   ],
   "require": {
     "php": "^7.1",
-    "prooph/common": "4.0.x-dev",
-    "prooph/event-store": "7.0.x-dev",
-    "prooph/pdo-event-store": "1.0.x-dev"
+    "prooph/common": "^4.0",
+    "prooph/event-store": "^7.0",
```

Comment:
There is no 7.0 final release yet. Use 7.0-dev, if that breaks it, we need to update micro-do.